src/index.html

```diff
@@ -13,6 +13,40 @@
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card card--image"
+      data-qa="card"
+    >
+      <div class="card__text">
+        <p class="card__text1">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__text2">Product code: 195434</p>
+        <div class="content">
+          <ul class="content__stars">
+            <li class="stars"></li>
```

Comment:
"stars" is not a _Block_, right? If it is an _Element_ of "content" the correct naming is `content__star`
Also, changing from stars to star, without the S, as the class refers to a single star and not to all of them